public/app/screens/main/sections/devices/device-state/style.scss

```diff
@@ -0,0 +1,21 @@
+@import "theme";
+
+.cc-device-state{
+  &__image {
+    @include grid-column(1);
```

Comment:
Your component assumes that is being used inside a `row`. Thus it's not reusable very well. Don't make any assumptions about the surrounding layout.